lib/requester/core-body-builder.js

```diff
@@ -108,6 +108,12 @@ formDataBodyReducer = function (data, param) {
         }
     }
 
+    // make sure `filename` param is sent for every file without `value`
+    // so that `filename=""` is added to content-disposition header in form data
+    if (param.type === 'file' && !param.value && typeof param.fileName !== 'string') {
+        param.fileName = E;
```

Comment:
should also set param.value='' to make sure it doesn't cause any problems down the line